internal/utils/config.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/spf13/viper"
 )
 
+var (
+	DEFAULT_ANON_KEY        string = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs"
+	DEFAULT_SERVER_ROLE_KEY string = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6InNlcnZpY2Vfcm9sZSJ9.vI9obAHOGyVVKa3pD--kJlyxp-Z2zV9UUMAhKpNLAcU"
+)
+
```

Comment:
Can use `const` instead of `var`. There's no plan to allow overriding these keys, so can drop the `DEFAULT_` for now. Also to follow Go's naming convention this could be `AnonKey` & `ServiceRoleKey` instead.